src/styles.css

```diff
@@ -0,0 +1,22 @@
+.form-container {
+  width: 30rem;
+  padding: 2rem;
+  position: absolute;
+  top: 10rem;
+  left: 20rem;
+  border: 0.1rem solid;
+}
+
+.btn-div {
+  text-align: center;
+  margin-top: 2rem;
+}
+
+.form-container::before {
```

Comment:
You don't have to put the form title in the exact same place as the wire frame. It's just an example.